dev/com.ibm.ws.org.apache.taglibs.standard/bnd.bnd

```diff
@@ -28,7 +28,7 @@ Export-Package: \
 
 Include-Resource: \
    @${repo;com.ibm.ws.javax.j2ee:jsp.tld.standard;1.2}!/META-INF/*.tld,\
-   @${repo;com.ibm.ws.org.apache:xalan;2.7.2}!/!(META-INF/*|org/apache/xalan/xsltc/*|java_cup/*|trax/*|org/apache/bcel/*|org/apache/regexp/*)
+   @${repo;com.ibm.ws.org.apache:xalan;2.7.3}!/!(META-INF/*|org/apache/xalan/xsltc/*|java_cup/*|trax/*|org/apache/bcel/*|org/apache/regexp/*)
```

Comment:
I'd still like to discuss if we need to continue to exclude these packages.

Reply:
In addition, we should check if we can stop using the "custom" com.ibm.ws JAR location and point directly to the xalan JAR everything else is using. It was updated to 2.7.2 here: https://github.com/OpenLiberty/open-liberty/pull/4630 and the 2.7.2 jars look identical. If that is the case then I don't see why we can't just point to the common 2.7.3 JAR. It has been 6 years since the 2.7.2 update and I'm not aware of any problems.